src/Concordium/Client/Output.hs

```diff
@@ -889,6 +889,17 @@ showEvent verbose ciM = \case
         verboseOrNothing $ printf "baker %s with account %s suspended" (show bID) (show acc)
     Types.BakerResumed bID acc ->
         verboseOrNothing $ printf "baker %s with account %s resumed" (show bID) (show acc)
+    Types.TokenModuleEvent tokenEvent ->
+        let Types.TokenEventDetails bss = Types._teDetails tokenEvent
+            invalidCBOR = printf "Could not decode token event details of token module event with token id %s and type %s as valid CBOR. The hex value of the event details is %s." (show $ Types._teSymbol tokenEvent) (show $ Types._teType tokenEvent) (show $ Types._teDetails tokenEvent)
+            bsl = BSL.fromStrict $ BSS.fromShort bss
+            eventDetails = case deserialiseFromBytes decodeString bsl of -- Try to decode the event details as a CBOR string
```

Comment:
I don't think in general the event details will be a string. Possibly `decodeValue` (from `Codec.CBOR.JSON`) could be used here and then render the value as JSON.